packages/driver/src/cy/commands/navigation.ts

```diff
@@ -1172,8 +1174,12 @@ export default (Commands, Cypress, cy, state, config) => {
           s.pending = Cypress.runner.countByTestState(s.tests, 'pending')
           s.numLogs = LogUtils.countLogsByTests(s.tests)
 
+          type ReporterRunRecords = ReporterRunState & {
```

Comment:
If the or condition is `any`, I don't think this type has any benefit unless we do some typechecking and casting. Might make sense in the reducer?